src/main/java/com/laborete/LaboreteAPI/posts/models/CreatePostDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.laborete.LaboreteAPI.posts.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+@Setter
+public class CreatePostDTO {
+
+    @JsonProperty("id")
+    private UUID id;
```

Comment:
Id is an internal field. When you create something you should not pass any id - it has to be generated under the hood.